server/core/config/raw/policies_test.go

```diff
@@ -75,17 +75,13 @@ func TestPolicySets_Validate(t *testing.T) {
 				PolicySets: []raw.PolicySet{
 					{
 						Name:   "policy-name-1",
+						Owner:  "owner1",
 						Path:   "rel/path/to/source",
 						Source: valid.LocalPolicySet,
 					},
 					{
-						Name: "policy-name-2",
-						Owners: raw.PolicyOwners{
-							Users: []string{
-								"john-doe",
-								"jane-doe",
-							},
-						},
+						Name:   "policy-name-2",
+						Owner:  "owner2",
```

Comment:
Why was this changed to a string? The original intention of using a struct was to allow us to add teams as a field so that we didn't have to make a bunch of these code changes.